docs/Updating-gene-and-gene_alias-tables.md

```diff
@@ -1,55 +1,57 @@
-```sql
-TRUNCATE TABLE mutation_event;
-TRUNCATE TABLE cna_event;
-```
+## MySQL steps
+Execute these steps in case you want to reset your database to the most recent genes list from NCBI.
 
-Another way of obtaining an empty database is starting a new MySQL database with the previous seed database.
+1- Start a new MySQL database with the previous seed database, which can be found on cBioPortal Datahub for [human](https://github.com/cBioPortal/datahub/tree/master/seedDB) and [mouse](https://github.com/cBioPortal/datahub/tree/master/seedDB_mouse).
 
 2- If DB engine supports foreign key (FK) constraints, e.g. InnoDB, drop constraints:
 ```sql
 ALTER TABLE cosmic_mutation
   DROP FOREIGN KEY cosmic_mutation_ibfk_1;
 
-ALTER TABLE sanger_cancer_census
-  DROP FOREIGN KEY sanger_cancer_census_ibfk_1;
-
 ALTER TABLE uniprot_id_mapping
   DROP FOREIGN KEY uniprot_id_mapping_ibfk_1;
 ```
 
 3- Empty tables `gene` and `gene_alias`
 ```sql
 TRUNCATE TABLE gene_alias;
-TRUNCATE TABLE gene;
+delete from gene;
 ```
 
 4- Restart cBioPortal (restart webserver) to clean-up any cached gene lists.
 
-5- You probably also want to update the gene lengths. This is dependent on the reference genome you want to use. Download .annotation.gtf.gz from the latest GENCODE release from http://www.gencodegenes.org/releases/current.html for hg38 / GRCh38. For the hg19 / GRCh37, click on 'Go to GRCh37 version of this release'.
+5- You probably also want to update the gene lengths. To do so, download .annotation.gtf.gz of the [latest GENCODE release for hg19 / GRCh37](http://www.gencodegenes.org/releases/26lift37.html) for human and [mm10 / GRCm38](http://www.gencodegenes.org/mouse_releases/current.html) for mouse.
 
 After downloading, go to your downloads directory, decompress the file and add it as an argument (--gtf) in the next step.
 
 6- To import gene data type the following commands when in the folder `<your_cbioportal_dir>/core/src/main/scripts`:
-
 ```
  export PORTAL_HOME=<your_cbioportal_dir>
-./importGenes.pl --genes <Homo_sapiens_gene_info.txt> --gtf <gencode.v25.annotation.gtf>
+./importGenes.pl --genes <ncbi_species.gene_info> --gtf <gencode.v25.annotation.gtf>
 ```
 
 7- :warning: Check the `gene` and `gene_alias` tables to verify that they are filled correctly.
+```sql
+SELECT count(*) FROM cbioportal.gene;
+SELECT count(*) FROM cbioportal.gene_alias;
+```
 
-8- Additionally, there are several other tables you may want to update now.
+8- Additionally, there are several other tables you may want to update now (only in human).
 
 1. Updating the pfam graphics if you want to see 3D structures, explained in [this document](https://github.com/oplantalech/cbioportal/blob/d74d7159ff6abba67739efa7f905e407b3961ed3/docs/Updating-pfam_graphics-table.md)
 
```

Comment:
@oplantalech Didn't notice this before, but we should probably have this point to a URL in `cbioportal` no?